src/components/Content/Content.styles.js

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+
+export const ContentWrapper = styled.div`
+    display: flex;
+    align-items: center;  
+    justify-content: center;
+`;
+
+export const CardContainer = styled.section`
+    display: block;    
+
+    @media (max-width: 485px) {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+    @media (max-width: 768px) {
+        display: flex; 
+        flex-direction: column;
+        justify-content: center;
+    }
+`;
+
+export const Card = styled.div`  
+  float: left;

```

Comment:
In terms of styling, I would prefer to use flexbox and probably add a defined height so all the cards have the same height.